dipy/tracking/tests/test_streamline.py

```diff
@@ -516,6 +517,63 @@ def test_unlist_relist_streamlines():
         assert_array_equal(streamlines[i], streamlines2[i])
 
 
+def test_deform_streamlines():
+    # streamlines needs to be a list of streamlines
+    A = np.array([[1, 2, 3], [1, 2, 3.]])
+    streamlines = [A for i in range(10)]
```

Comment:
What happens if your streamlines don't have all the same number of points? You could use one of the global variables defined at the top of this test file (e.g. `streamlines` define at line 152).

Reply:
am now testing with global streamlines